worldedit-core/src/main/java/com/fastasyncworldedit/core/extension/factory/parser/transform/OffsetTransformParser.java

```diff
@@ -38,7 +38,7 @@ public OffsetTransformParser(WorldEdit worldEdit) {
     @Override
     protected ResettableExtent parseFromInput(@NotNull String[] arguments, ParserContext context) throws InputParseException {
```

Comment:
Looking on the import, we're using Jetbrains' annotations here, but (FA)WE uses javax. Is there a specific reason to use Jetbrains annotations here? Otherwise I'd vouch for exchanging it for sake of consistency.